miso-web/src/main/webapp/pages/bulkEditLibraries.jsp

```diff
@@ -78,15 +78,15 @@
        <p>The following rows failed to save:</p>
        <p id="errorMessages"></p>
      </div>
-   
+
    <div id="hotContainer"></div>
- 
+
  </div>
-  
+
   <script type="text/javascript">
     jQuery(document).ready(function () {
       Library.hot.librariesJSON = ${librariesJSON};
-      Library.hot.librariesJSON = Hot.sortByProperty(Library.hot.librariesJSON, 'parentSampleAlias');
```

Comment:
Please wrap the `Hot.sortByProperty` line if an `if Hot.detailedSample` check (and move the `Hot.detailedSample` assignment up above this line) instead of deleting it, as we will still want to use this sorting in our lab.